google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -837,18 +880,22 @@ def _download_table_bqstorage(
     if "@" in table.table_id:
         raise ValueError("Reading from a specific snapshot is not currently supported.")
 
-    requested_streams = 1 if preserve_order else 0
+    # Compares preserve_order vs max_stream_count to determine how many
+    # streams to use.
```

Comment:
nit: this comment doesn't add all that much to the readability of the code, so I would just drop it (preserve_order and max_stream_count are being used as arguments to a well named function, which implies to the reader they are used to determine the number of requested streams).

```suggestion
```